english/check-phone/check-phone.md

```diff
@@ -0,0 +1,7 @@
+You get a notification that someone has texted you, but you don't 
```

Comment:
Sorry, Casey does not understand links that span two lines, as the link that leads to this page does. This page actually is reachable.